frontend/src/_services/organization.service.js

```diff
@@ -11,9 +12,21 @@ export const organizationService = {
   editOrganizationConfigs,
 };
 
-function getUsers() {
+function getUsers(page, options) {
   const requestOptions = { method: 'GET', headers: authHeader() };
-  return fetch(`${config.apiUrl}/organizations/users`, requestOptions).then(handleResponse);
+  return fetch(
+    `${config.apiUrl}/organizations/users?page=${page}&${options?.email ? `email=${options.email}` : ''}&${
+      options?.firstName ? `firstName=${options.firstName}` : ''
+    }&${options?.lastName ? `lastName=${options.lastName}` : ''}`,
```

Comment:
```suggestion
    `${config.apiUrl}/organizations/users?page=${page}${options?.email ? `&email=${options.email}` : ''}${
      options?.firstName ? `&firstName=${options.firstName}` : ''
    }${options?.lastName ? `&lastName=${options.lastName}` : ''}`,
```

Reply:
I would like to add there is a [`query-string`](https://github.com/sindresorhus/query-string#falsy-values) package available to use in the fe repo, 

```js
import queryString from 'query-string';
...
const { firstName, lastName, email } = options;
const params = queryString.stringify({ page, firstName, lastName, email });
fetch(`${config.apiUrl}/organizations/users?${query}`)
...
```